### ISSUE TITLE
SantaGate

### ISSUE THREAD

**Issue by mikeblakeuk** [fcapano/Edit-This-Cookie#268]
Please can you give me the option to have the Santa Hat all year. I love it.
https://github.com/ETCExtensions/Edit-This-Cookie/blob/master/img/cookie_xmas_38x38.png

Thanks

re: https://github.com/microsoft/vscode/issues/87440#issue-540437717

**Comment by viktor-ku** [comment]
yes, please, me too, I want to have a santa hat all year, I think it's cute :santa:
